src/Core.UnitTests/SystemAbstractions/DefaultCurrentTimeProviderTests.cs

```diff
@@ -30,25 +28,29 @@ namespace SonarLint.VisualStudio.Core.UnitTests.SystemAbstractions
     public class DefaultCurrentTimeProviderTests
     {
         [TestMethod]
-        public void Instance_ReturnsCurrentTime()
+        public void MefCtor_IsSingleton()
         {
-            var now = DefaultCurrentTimeProvider.Instance.Now;
-            now.Should().BeOnOrBefore(DateTimeOffset.Now);
+            MefTestHelpers.CheckIsSingletonMefComponent<DefaultCurrentTimeProvider>();
         }
 
+        [TestMethod]
+        public void MefCtor_CheckIsExported()
+        {
+            MefTestHelpers.CheckTypeCanBeImported<DefaultCurrentTimeProvider, ICurrentTimeProvider>();
+        }
 
         [TestMethod]
-        public void Instance_ReturnsLocalTimeZone()
+        public void Now_ReturnsCurrentTime()
         {
-            DefaultCurrentTimeProvider.Instance.LocalTimeZone.Should().Be(TimeZoneInfo.Local);
+            ICurrentTimeProvider currentTimeProvider = new DefaultCurrentTimeProvider();
+            currentTimeProvider.Now.Should().BeOnOrBefore(DateTimeOffset.Now);
```

Comment:
I would use a delta comparison here or create two datetimes, one before and one after the `.Now` call